ssr/render.js

```diff
@@ -176,7 +177,7 @@ export default function render(
     $('meta[name="description"]').attr("content", pageDescription);
   }
 
-  if ((doc && !doc.noIndexing) || pageNotFound) {
+  if (ALWAYS_NO_ROBOTS || (doc && doc.noIndexing) || pageNotFound) {
```

Comment:
On MDN currently, for the case when we want to index a document, we always explicitly add a `<meta name="robots" content="index, follow">` tag, but with this code we don't, so we're counting on the default behavior of the various crawlers (Google, Bing, etc.) being `index, follow`. I'm not sure about that. For example, I found [this](https://yoast.com/help/why-does-yoast-seo-output-index-follow-in-the-meta-robots-tag/), which made me doubt that we can count on the default behaviors, and [this](https://moz.com/learn/seo/robots-meta-directives) which suggests that the default is actually `index`, not `index, follow`.

I think we should explicitly add the `<meta name="robots" content="index, follow">` tag.

Reply:
I'll take it!